test/slackutil/message_read.go

```diff
@@ -53,6 +53,8 @@ func NewReadClient(userName, tokenPath string) (ReadOperations, error) {
 	}, nil
 }
 
+// MessageHistory returns the list of messages sent by the user in the given
+// channel since the given startTime.
 func (c *readClient) MessageHistory(channel string, startTime time.Time) ([]string, error) {
 	u, _ := url.Parse(conversationHistoryURL)
 	q := u.Query()
```

Comment:
does it support query with the user name, so there is no need to filter on the receiving side?

Reply:
According to the API doc - https://api.slack.com/methods/conversations.history#response, it doesn't. I don't remember why I added `username` as one param, but it should not take any effect so just removed it. 
Also changed the alert logic a bit to put the min alert interval restriction for each test.